osu.Game/Overlays/SkinEditor/SkinEditor.cs

```diff
@@ -416,7 +416,7 @@ private void skinChanged()
             // Since skinChanged is scheduled, it has a chance to be called right after PopOut is called, therefore leasedSkinInfo may be null here.
             if (leasedSkinInfo != null)
                 leasedSkinInfo.Value = skins.GetMutableSkin();
-            else
+            else if (!skins.CurrentSkinInfo.Disabled)
```

Comment:
I *really* don't like this. The mutable skin is required to use the editor... so if this somehow triggers it's going to manifest as a bug.

If we can't guarantee that the skin can be changed, then this solution needs to be reconsidered.

Reply:
The proposal to disable the dropdown was a throwaway suggestion to begin with, with the idea that it would be very quick and simple to make work, so if even this isn't simple enough to make work well I'd say this PR should be closed at this stage.

Reply:
> I _really_ don't like this. The mutable skin is required to use the editor... so if this somehow triggers it's going to manifest as a bug.
> 
> If we can't guarantee that the skin can be changed, then this solution needs to be reconsidered.

This specific condition is caused by a specific scenario wherein multiple skin editor instances are loaded at once (which is only a thing in test scenes). I could add an inline comment if that helps.

Reply:
@peppy I'm also interested in your thoughts about the other solution I proposed (see last paragraph of [OP](https://github.com/ppy/osu/pull/27013#issue-2116967038)).